internal/npm_install/test/generate_build_file.spec.js

```diff
@@ -1,7 +1,7 @@
-const check = require('./check');
+const {check, files} = require('./check');
 
 describe('build file generator', () => {
-  it('should produce a BUILD file from the node_modules file structure', () => {
-    check();
+  it('should produce BUILD files from the node_modules file structure', () => {
+    files.forEach(file => check(file));
```

Comment:
nit: it might be better to generate `it` blocks like

```
describe('', () => {
  files.forEach(file => {
    it(`should produce a BUILD file for ${file}`, () => {
      check(file);
    });
  });
});

Reply:
👍